proto/viam/app/build/v1/build.proto

```diff
@@ -83,3 +99,74 @@ message ListJobsResponse {
   // jobs is ordered by (build start time, alphabetical platform).
   repeated JobInfo jobs = 1;
 }
+
+message RepoLink {
+  string app_link_id = 1;
```

Comment:
Is this an Id from GH?

Reply:
if it is, can we call oauth_app_link id or something? so it's a bit more clear where it comes from

Reply:
yup way clearer, done in 33c41a0